### ISSUE TITLE
Unable to run wdio V5 test on CircleCI

### ISSUE THREAD

**Issue by Jarv1193** [webdriverio/webdriverio#4931]
**Environment (please complete the following information):**
 - **WebdriverIO version:** 5.16.11
 - **Mode:** Standalone mode 
 - **Node.js version:**  8.11.2
 - **NPM version:**  6.5.0
 - **Browser name and version:** Firefox 71
 - **Platform name and version:** Debian GNU/Linux 9 \n \l
 - **Additional wdio packages used (if applicable):**  @wdio/dot-reporter, @wdio/selenium-standalone service

**Config of WebdriverIO**

    const screenshotLocation = './test/functional/artifacts/screenshots/';
    const baselineLocation = './test/functional/artifacts/baseline/';
    const seleniumStandaloneConfigLocation = './selenium-standalone-config.js';
    const path = require('path');

    exports.config = {

    runner: 'local',

    specs: [
        './test/functional/specs/**/*-spec.js'
    ],

    exclude: [
        // 'path/to/excluded/files'
    ],
 
    maxInstances: 10,
    
    capabilities: [{
        browserName: 'firefox',
        'acceptInsecureCerts': true,
    }],
    
    logLevel: 'error',

    bail: 0,

    baseUrl: 'http://localhost',

    waitforTimeout: 10000,

    connectionRetryCount: 3,

    services: ['selenium-standalone', 
        ['image-comparison', 
        // The options
        {
            // Some options, see the docs for more
            baselineFolder: baselineLocation,
            formatImageName: '{tag}-{logName}-{width}x{height}',
            screenshotPath: screenshotLocation,
            savePerInstance: true,
            autoSaveBaseline: true,
            blockOutStatusBar: true,
            blockOutToolBar: true,
        }],
    ],

    framework: 'jasmine',

    reporters: ['dot','junit'],
 
    jasmineNodeOpts: {
       defaultTimeoutInterval: (30 * 60 * 1000), 
       random: true,
       cleanStack: true,
       expectationResultHandler: function(passed, assertion) {
           if(!passed) {
               let fileName = screenshotLocation + `assertion_failure_${new Date().getTime()}.png`;
               let absoluteFile = path.resolve(fileName);
               console.log(`Failure screenshot saved at: ${absoluteFile}`);
               browser.saveScreenshot(fileName);
           }
       }
    },
    
    seleniumLogs: './test/functional/artifacts/logs/selenium.txt',
    seleniumArgs: require(seleniumStandaloneConfigLocation),
    seleniumInstallArgs: require(seleniumStandaloneConfigLocation),
    }

**Describe the bug**
Basically I can't run my wdio test on CircleCi

**To Reproduce**
CircleCI with the environment mentioned 

**Expected behavior**
Get test executed on CircleCI

**Additional context**
![image](https://user-images.githubusercontent.com/37460773/71683028-9d6bad00-2d4e-11ea-91b2-334dabb2553d.png)

**Comment by mgrybyk** [comment]
Please see example here https://github.com/mgrybyk/wdio-jasmine-boilerplate

**Comment by Jarv1193** [comment]
@mgrybyk still getting the error
`[0-1] 2020-01-02T18:53:50.424Z ERROR webdriver: Request failed due to unknown error: invalid argument: can't kill an exited process
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:32:19.891Z'
System info: host: '0194c87226a5', ip: '172.30.0.3', os.name: 'Linux', os.arch: 'amd64', os.version: '4.15.0-1052-aws', java.version: '1.8.0_232'
Driver info: driver.version: unknown
remote stacktrace: 
    at getErrorFromResponseBody (/home/circleci/repo/node_modules/webdriver/build/utils.js:124:10)
    at Request._callback (/home/circleci/repo/node_modules/webdriver/build/request.js:110:64)
    at Request.self.callback (/home/circleci/repo/node_modules/webdriver/node_modules/request/request.js:185:22)
    at emitTwo (events.js:126:13)
    at Request.emit (events.js:214:7)
    at Request.<anonymous> (/home/circleci/repo/node_modules/webdriver/node_modules/request/request.js:1161:10)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at IncomingMessage.<anonymous> (/home/circleci/repo/node_modules/webdriver/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:313:30)`

**Comment by christian-bromann** [comment]
> still getting the error

You need to provide more logs (e.g. driver and wdio logs)
